python/ray/serve/pipeline/tests/test_deployment_node.py

```diff
@@ -43,7 +43,7 @@ async def get(self):
         return self.i
 
 
-@pytest.mark.asyncio
+@pytest.mark.skip(reason="async handle not enabled yet")
```

Comment:
I thought async handle needs to be implemented for performance?

Reply:
yes but not in this PR yet